.travis.yml

```diff
@@ -18,22 +18,26 @@ addons:
 install:
   - ""
 
+# JDKs used in builds must be supported by Tycho. Check https://wiki.eclipse.org/Tycho/Release_Notes to find supported versions.
```

Comment:
please wrap long line to follow limit of 100 on line.
now it cause horizontal scrolling - not good for review.

Reply:
Done. Don't easily notice that on 4K :)